### PR TITLE
Update BTF style map pinning property

### PR DESCRIPTION
**Pull request by sanfern**
This PR updates BTF style map pinning property to xdp-root, RL and CL programs.

### CODE REVIEW

File: connection-limit/connection_limit.bpf.c [hunk 1]
```diff
@@ -62,23 +62,26 @@ struct {
...
+    __uint(pinning, LIBBPF_PIN_BY_NAME);
 } cl_tcp_conns SEC(".maps");
 
 /* Maintains concurrent connection sockets */
 struct {
     __uint(type, BPF_MAP_TYPE_HASH);
     __uint(key_size, sizeof(uint64_t));
     __uint(value_size, sizeof(uint32_t));
-   __uint(max_entries, 30000);
+    __uint(max_entries, 30000);
```
Comment (aka320) [inline connection-limit/connection_limit.bpf.c:83]: Can we use MACROs for the max_entries instead of hard-coded value for all the maps ?

Comment (sanfern) [reply connection-limit/connection_limit.bpf.c:83]: It's not repeating, what is the advantage of macro here?

**Review by jaysheth2 — APPROVED** [review]
LGTM

**Comment by aka320** [comment]
LGTM

**Review by aka320 — APPROVED** [review]

**Pull request merged**